conf/fusesoc-configs/earlgrey.yml

```diff
@@ -0,0 +1,9 @@
+name: earlgrey
+description: Earlgrey design from opentitan
+top_module: top_earlgrey_nexysvideo
+tags: earlgrey uvm
+path: &path third_party/cores/opentitan
+command: ["fusesoc", "--cores-root", *path, "run", "--target=sim", "lowrisc:systems:top_earlgrey:0.1"]
```

Comment:
Could we make it a string and split it to a list in the python script if it's necessary? That would make the config much cleaner.